.env.example

```diff
@@ -1,8 +1,10 @@
-NODE_ENV=development
-PORT=4000
-JWT_SECRET=0a6b944d-d2fb-46fc-a85e-0295c986cd9f
-PG_DB=api
-PG_PORT=5432
-PG_HOST=localhost
-PG_USER=amida
-PG_PASSWD=''
\ No newline at end of file
+#NODE_ENV=development
+#PORT=4000
+#API_VERSION=1.0
+#JWT_SECRET=helloworld
+#PG_DB=api
+#PG_TEST_DB=api-test
+#PG_PORT=5432
+#PG_HOST=localhost
+#PG_USER=postgres
+#PG_PASSWD=password
```

Comment:
This might be out of scope, but we might want to add in support for encrypted connections to postgres. This has already been done for the microservices, see this PR https://github.com/amida-tech/amida-auth-microservice/pull/46